packages/components/molecules/f-restaurant-card/src/components/subcomponents/RestaurantRating/_tests/RestaurantRating.test.js

```diff
@@ -189,7 +189,7 @@ describe('RestaurantRating component', () => {
             expect(countMessage.exists()).toBe(true);
 
             expect(ratingsMeanElement.text()).toStrictEqual('5.0');
-            expect(countMessage.text()).toStrictEqual('200+');
+            expect(countMessage.text()).toStrictEqual('200');
```

Comment:
I think the previous test should still be included, and we probably need a few more to test when it's less, equal to and when threshold is/isn't provided